### ISSUE TITLE
README, provenance information, and conversion scripts?

### ISSUE THREAD

**Issue by jchodera** [brycestx/perses_jacs_systems#1]
Thanks for putting this online, @brycestx!

It would be great if you could also throw up a `README.md` noting how the JACS SI material was processed to produce these files, as well as the script you used to process the data! This would be critical for putting together a paper with benchmark data on this prepped set.

**Comment by brycekallen** [comment]
I agree, I've been meaning to get around to this. Let me take care of it now
